xds/internal/balancer/priority/balancer_test.go

```diff
@@ -617,7 +632,14 @@ func (s) TestPriority_HigherReadyCloseAllLower(t *testing.T) {
 	//
 	// With localities caching, the lower priorities are closed after a timeout,
 	// in goroutines. The order is no longer guaranteed.
-	scToShutdown := []balancer.SubConn{<-cc.ShutdownSubConnCh, <-cc.ShutdownSubConnCh}
+	firstSC := <-cc.ShutdownSubConnCh
+	secondSC := <-cc.ShutdownSubConnCh
+	// The same SubConn is closed by balancergroup, gracefulswitch and
+	// pickfirstleaf when they are closed. Remove duplicate events.
```

Comment:
This is just temporary, though, right?  Once PF is the universal leaf policy, nothing else should need to shut down SubConns (the last bullet point in #6472).  So a TODO here would be good.

Reply:
Added a TODO everywhere.